### DISCUSSION TITLE
Is it possible to open only saved tabs on CotEditor startup?

### DISCUSSION THREAD

**Discussion by johnmapeson** [coteditor/CotEditor#1756] [Q&A] [answered]
I have a CotEditor window with 4 tabs. Two of them are "real" files (foo.txt and bar.txt on my Desktop) and two were created using Command-N but never saved, and I use them as temporary buffers. Is there a setting or a combination of settings so that when I press Command-Q, the window is immediately closed without prompting me to save anything, and when I open CotEditor the next time, only the first two tabs are opened, that is, the two tabs that represent "real" files?

**Comment by 1024jp** [comment, marked as answer]
No. CotEditor doesn't have such an option.
